src/pages/index.js

```diff
@@ -58,32 +119,47 @@ const defautlFormConfig = {
 
 const editFormValidator = new FormValidator(defautlFormConfig, profilePencilModal);
 const addCardFormValidator = new FormValidator(defautlFormConfig, addCardModal);
+const editAvatarFormValidator = new FormValidator(defautlFormConfig, profileAvatarModal);
 
 editFormValidator.enableValidation();
 addCardFormValidator.enableValidation();
+editAvatarFormValidator.enableValidation();
 
 // Render Initial Cards from Section class
-const cardSection = new Section({
-  items: initialCards,
-  renderer: (cardData) => {
-     const cardElement = createCard(cardData);
-    cardSection.addItem(cardElement);
-  }
-}, '.cards__content');
 
-cardSection.renderItems();
-
-// UserInfo 
+let cardSection
+
+Promise.all([api.getCurrentUser(), api.getInitialCards()])
+  .then(([data, InitialCards]) => {
```

Comment:
`InitialCards`

Every variable in the project should be in `camelCase` (in **lower case** if it's not a class or component)

Reply:
Fixed to initialCards